awx/main/tests/functional/test_inventory_source_injectors.py

```diff
@@ -7,9 +7,10 @@
 
 from awx.main.tasks.jobs import RunInventoryUpdate
 from awx.main.models import InventorySource, Credential, CredentialType, UnifiedJob, ExecutionEnvironment
-from awx.main.constants import CLOUD_PROVIDERS, STANDARD_INVENTORY_UPDATE_ENV
+from awx.main.constants import STANDARD_INVENTORY_UPDATE_ENV
 from awx.main.tests import data
 from awx.main.utils.execution_environments import to_container_path
+from awx.main.utils.plugins import CLOUD_PROVIDERS
```

Comment:
```suggestion
from awx.main.utils.plugins import discover_available_cloud_provider_plugin_names
```

And below, you will need to inject this call into a fixture, then inject that fixture into the test and use `pytest-subtests` inside that test to emulate what parametrize did externally.